### PR TITLE
Add comment for the release

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
Adding Kafka Trigger/Output documentation for the auto generated documentation.

### CODE REVIEW

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java [hunk 1]
```diff
@@ -14,13 +14,43 @@
 import java.lang.annotation.Target;
 
 /**
- * <p>Annotation for Kafka output bindings</p>
+ * <p>
+ * Place this on a parameter whose value would be written to Kafka. The parameter type should be
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java:18]: published

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java [hunk 2]
```diff
@@ -14,13 +14,43 @@
...
+ *     final String message,
+ *    {@literal @}KafkaOutput(name = "event", topic = "users", brokerList="broker:29092") OutputBinding&lt;String&lt; output,
+ *    final ExecutionContext context) {
+ *     context.getLogger().info("Message:" + message);
+ *     output.setValue(message);
+ *     return "{ \"id\": \"" + System.currentTimeMillis() + "\", \"description\": \"" + message + "\" }";
+ * }
+ * </pre>
+ * 
+ * @since 1.0.0
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java:46]: 1.4.0

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java [hunk 3]
```diff
@@ -36,57 +66,76 @@
     String dataType() default "";
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java:66]: Can we add comment?

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java [hunk 4]
```diff
@@ -95,37 +144,49 @@
...
+     * @return The protocol.
      */
     BrokerProtocol protocol() default BrokerProtocol.NOTSET;
 
     /**
      * Path to client's private key (PEM) used for authentication.
      * Default ""
      * ssl.key.location in librdkafka
+     * 
+     * @return The ssl.key.location.
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java:165]: Can we remove the dots or add dots to ssl.ca.location?

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java (hunk 1 above)
Comment (TsuyoshiUshio) [reply src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java:18]: Done!

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java (hunk 2 above)
Comment (TsuyoshiUshio) [reply src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java:46]: Done

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java [hunk 5]
```diff
@@ -95,37 +144,49 @@
...
+     * 
+     * @return The ssl ca location.
      */
     String sslCaLocation() default "";
 
     /**
      * Path to client's certificate.
      * ssl.certificate.location in librdkafka
+     * 
+     * @return The client certificate.
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java:181]: Can we change it to The ssl client certification location?

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java (hunk 3 above)
Comment (TsuyoshiUshio) [reply src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java:66]: It looks have a comment. Could you double check it?

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java (hunk 4 above)
Comment (TsuyoshiUshio) [reply src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java]: Done

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java (hunk 5 above)
Comment (TsuyoshiUshio) [reply src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java]: Done

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java [hunk 6]
```diff
@@ -16,7 +16,37 @@
...
+ *
+ * <p>
+ * The following example shows a Java function that is invoked when messages are consumed with
+ * the specified topic, brokerList, and consumerGroup on a Kafka cluster.
+ * </p>
+ *
+ * <pre>
+ * {@literal @}FunctionName("KafkaTrigger-Java")
+ * public void run(
+ *    {@literal @}KafkaTrigger(name = "kafkaTrigger",
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java:38]: Should we define datatype?

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java [hunk 7]
```diff
@@ -16,7 +16,37 @@
...
+ *                      brokerList="broker:29092",
+ *                      consumerGroup="functions")
+ *                      List&lt;Map&lt;String, String&gt;&gt; kafkaEventData,
+ *     final ExecutionContext context
+ * ) {
+ *     context.getLogger().info(kafkaEventData);
+ * }
+ * </pre>
+ *
+ * @since 1.0.0
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java]: 1.4.0

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java (hunk 6 above)
Comment (TsuyoshiUshio) [reply src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java:38]: No. dataType has default value. :)

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java (hunk 7 above)
Comment (TsuyoshiUshio) [reply src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java]: Done!

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java [hunk 8]
```diff
@@ -29,49 +59,63 @@
     String name();
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java:59]: Add comment

Comment (TsuyoshiUshio) [reply src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java:59]: It looks having a comment. The style is wrong?

File: src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java [hunk 9]
```diff
@@ -24,7 +24,7 @@
  * <ul>
  *     <li>Any native Java types such as int, String, byte[]</li>
  *     <li>Nullable values using Optional&lt;T&gt;</li>
- *     <li>Any POJO type</li>
+ *     <li>Any POJO type for Cardinality.One</li>
```
Comment (amamounelsayed) [inline src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java:27]: What about Any POJO type, currently supported only for Cardinality.One

Comment (TsuyoshiUshio) [reply src/main/java/com/microsoft/azure/functions/annotation/KafkaTrigger.java:27]: Very good one! Thanks!

**Pull request merged**